hera_cal/vis_clean.py

```diff
@@ -378,7 +378,7 @@ def vis_clean(self, keys=None, data=None, flags=None, wgts=None, ax='freq', hori
 
         # iterate over keys
         for k in keys:
-            if k in self.clean_model and overwrite is False:
+            if k in getattr(self, clean_model) and overwrite is False:
```

Comment:
For clarity, I would rename these variables something like "clean_model_name"... right now it's confusing that they are strings rather than the model/flags/resid themselves.

Reply:
ok, I think I'll just assign them as the datacontainers, which is probably easier to read anyways